Included a link to regex flag descriptions in the regexp document (#31236)
mdn/content: files/en-us/web/javascript/reference/global_objects/regexp/flags/index.md

## See also

> - [A table of regular expression flags and their description](/en-US/docs/Web/JavaScript/Guide/Regular_expressions#advanced_searching_with_flags)
> - [Polyfill of `RegExp.prototype.flags` in `core-js`](https://github.com/zloirock/core-js#ecmascript-string-and-regexp)

Reviewer: ```suggestion
- [Polyfill of `RegExp.prototype.flags` in `core-js`](https://github.com/zloirock/core-js#ecmascript-string-and-regexp)
- [Advanced searching with flags](/en-US/docs/Web/JavaScript/Guide/Regular_expressions#advanced_searching_with_flags) in the Regular expressions guide
```

(Also the link is already in the intro paragraph)

Author: > (Also the link is already in the intro paragraph)

Ah. I think I missed that because I expected to find that table with in the document. Should this PR be closed without merging?

Reviewer: No, a see also is certainly better!
